ARExperiment/ARExperiment/UIViewController+Helpers.swift

```diff
@@ -0,0 +1,12 @@
+import UIKit
+
+extension UIViewController {
+    func viewBackgroundColor(to color: UIColor) {
+        self.view.backgroundColor = color
+    }
+
+    func navigationBar(with color: UIColor, and name: String) {
```

Comment:
I realise that I caught you up with my comment about fluent methods earlier, but be careful not to make them so fluent they aren't clear! 

In this case, it is hard to tell as a user what the types are here without clicking through to see the implementation, or just guessing. For example, `navigationBar(with:, and:)` is ambiguous as to what `with` and `and` actually relate to.

The above method (`viewBackgroundColor(to:)`) is clear because it is obvious what we are setting - a background colour. there's no similar context in `navigationBar(with:, and:)`.

If you read the [Swift API design guidelines](https://swift.org/documentation/api-design-guidelines/#naming), I'd point that this could be improved by the points:
- "Include all the words needed to avoid ambiguity" 
- "Compensate for weak type information"


```suggestion
    func styleNavigationBar(withColor color: UIColor, title title: String) {
```